db/types/base.py

```diff
@@ -9,3 +75,11 @@
 
 def get_qualified_name(name):
     return ".".join([preparer.quote_schema(SCHEMA), name])
+
+
+def get_installed_types(engine):
+    return engine.dialect.ischema_names
```

Comment:
This name could lead to confusion, since the  types are not necessarily installed at the DB level, but only need to be available in the engine.  Also, consider using this function inside of `db.types.alteration.get_supported_alter_column_types` (in case we change where we store the info pulled from `ischema_names` in the future).

Reply:
Fixed, I updated most places to use this instead of `engine.dialect.ischema_names` directly. I also changed the name to `get_available_types`.